<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Home</title>

<script src="site_libs/header-attrs-2.23/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/united.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>



<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Cocaine IVSA mDA Neurons RNASequencing Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="DAvsnDA.html">Dopamine vs Non-Dopamine Analysis</a>
</li>
<li>
  <a href="about.html">About Our Lab</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Home</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-07-18
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong>
<code>CocaineIVSA_RNASequencing_mDANeurons/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20240716code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20240716)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20240716code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20240716)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it
difficult for you and others to run your code on a different machine.
Change the absolute path(s) below to the suggested relative path(s) to
make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
/home/avm27/Documents/Raw_Sequencing_Data/CIVSA2/CocaineIVSA_RNASequencing_mDANeurons
</td>
<td style="text-align:left;">
.
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomavm27CocaineIVSARNASequencingmDANeuronstreee59f23d031a850a2b040fdb473e76c1da665ec8ctargetblanke59f23da">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/tree/e59f23d031a850a2b040fdb473e76c1da665ec8c" target="_blank">e59f23d</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomavm27CocaineIVSARNASequencingmDANeuronstreee59f23d031a850a2b040fdb473e76c1da665ec8ctargetblanke59f23da"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/tree/e59f23d031a850a2b040fdb473e76c1da665ec8c" target="_blank">e59f23d</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  VennDiagram.2024-07-18_13-14-03.log
    Untracked:  VennDiagram.2024-07-18_13-17-53.log

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/DAvsnDA.Rmd</code>) and HTML
(<code>docs/DAvsnDA.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/e59f23d031a850a2b040fdb473e76c1da665ec8c/analysis/DAvsnDA.Rmd" target="_blank">e59f23d</a>
</td>
<td>
avm27
</td>
<td>
2024-07-18
</td>
<td>
Updating
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/7c1df5e6061f4671761a1740a2132edfc3958596/docs/DAvsnDA.html" target="_blank">7c1df5e</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/c31538bbb33383ec16632a62b439809eb31d4b2e/docs/DAvsnDA.html" target="_blank">c31538b</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/bfc97c14694a0cfb3bb5699555e3e157ce3f3035/docs/DAvsnDA.html" target="_blank">bfc97c1</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/188423a2a92d3cb00490e955271eb63483f3b0b9/analysis/DAvsnDA.Rmd" target="_blank">188423a</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
<td>
Testing
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/1ee1d086cd39bf3115343f820fe5f9cd5de1d6a1/docs/DAvsnDA.html" target="_blank">1ee1d08</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/a6b1eb649836114ff2c32dbcc880f39c771211d8/analysis/DAvsnDA.Rmd" target="_blank">a6b1eb6</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
<td>
Testing
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p><font size="16"> <strong>Purification of a midbrain dopaminergic
population</strong> </font></p>
<div id="deseq2-analysis-dopamine-vs-nondopamine" class="section level1"
number="1">
<h1><span class="header-section-number">1</span> DESeq2 Analysis –
Dopamine vs nonDopamine</h1>
<p>Building DESeq2 input files from gene count matrices output by
StringTie.</p>
<p>Running DeSeq2 analysis with factor level of Dopamine vs
Non-Dopamine</p>
</div>
<div id="plot-estimates-and-gather-results" class="section level1"
number="2">
<h1><span class="header-section-number">2</span> Plot Estimates and
Gather Results</h1>
<div id="gather-results" class="section level2 tabset" number="2.1">
<h2 class="tabset"><span class="header-section-number">2.1</span> Gather
Results</h2>
<p>These results include overview for each result file and dispersion
estimates based on count values.</p>
<div id="dopamine-vs-nondopamine" class="section level3" number="2.1.1">
<h3><span class="header-section-number">2.1.1</span> Dopamine vs
nonDopamine</h3>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:1000px; ">
<table class="table table-striped table-hover table-condensed table-responsive" style="font-size: 10px; font-family: Arial; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
DA vs nDA Results Overview
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
baseMean
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
log2FoldChange
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
lfcSE
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
stat
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
padj
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Mrpl15|Mrpl15
</td>
<td style="text-align:left;">
1701.36388
</td>
<td style="text-align:left;">
0.0524999
</td>
<td style="text-align:left;">
0.3780871
</td>
<td style="text-align:left;">
0.1388567
</td>
<td style="text-align:left;">
0.8895634
</td>
<td style="text-align:left;">
0.9540289
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm39587|Gm39587
</td>
<td style="text-align:left;">
30.97149
</td>
<td style="text-align:left;">
1.2514060
</td>
<td style="text-align:left;">
1.8548426
</td>
<td style="text-align:left;">
0.6746696
</td>
<td style="text-align:left;">
0.4998857
</td>
<td style="text-align:left;">
0.7145863
</td>
</tr>
<tr>
<td style="text-align:left;">
Lypla1|Lypla1
</td>
<td style="text-align:left;">
812.46262
</td>
<td style="text-align:left;">
0.7213270
</td>
<td style="text-align:left;">
1.0808215
</td>
<td style="text-align:left;">
0.6673878
</td>
<td style="text-align:left;">
0.5045245
</td>
<td style="text-align:left;">
0.7183553
</td>
</tr>
<tr>
<td style="text-align:left;">
Xkr4|Xkr4
</td>
<td style="text-align:left;">
3572.74595
</td>
<td style="text-align:left;">
0.3489398
</td>
<td style="text-align:left;">
0.3517840
</td>
<td style="text-align:left;">
0.9919149
</td>
<td style="text-align:left;">
0.3212390
</td>
<td style="text-align:left;">
0.5439401
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm39585|Gm39585
</td>
<td style="text-align:left;">
7249.50804
</td>
<td style="text-align:left;">
-1.2500871
</td>
<td style="text-align:left;">
0.3299238
</td>
<td style="text-align:left;">
-3.7890174
</td>
<td style="text-align:left;">
0.0001512
</td>
<td style="text-align:left;">
0.0016620
</td>
</tr>
<tr>
<td style="text-align:left;">
Tcea1|Tcea1
</td>
<td style="text-align:left;">
2301.25708
</td>
<td style="text-align:left;">
0.8631673
</td>
<td style="text-align:left;">
0.3453679
</td>
<td style="text-align:left;">
2.4992689
</td>
<td style="text-align:left;">
0.0124450
</td>
<td style="text-align:left;">
0.0589906
</td>
</tr>
<tr>
<td style="text-align:left;">
Rgs20|Rgs20
</td>
<td style="text-align:left;">
427.37294
</td>
<td style="text-align:left;">
0.0119118
</td>
<td style="text-align:left;">
0.8629151
</td>
<td style="text-align:left;">
0.0138042
</td>
<td style="text-align:left;">
0.9889862
</td>
<td style="text-align:left;">
0.9957082
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm16041|Gm16041
</td>
<td style="text-align:left;">
54.96913
</td>
<td style="text-align:left;">
-9.2135915
</td>
<td style="text-align:left;">
1.5556113
</td>
<td style="text-align:left;">
-5.9228107
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000001
</td>
</tr>
<tr>
<td style="text-align:left;">
Atp6v1h|Atp6v1h
</td>
<td style="text-align:left;">
8199.89727
</td>
<td style="text-align:left;">
1.5325294
</td>
<td style="text-align:left;">
0.1984866
</td>
<td style="text-align:left;">
7.7210724
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
LOC108167788|LOC108167788
</td>
<td style="text-align:left;">
19.60403
</td>
<td style="text-align:left;">
-0.5421843
</td>
<td style="text-align:left;">
1.9713087
</td>
<td style="text-align:left;">
-0.2750377
</td>
<td style="text-align:left;">
0.7832873
</td>
<td style="text-align:left;">
0.9037852
</td>
</tr>
<tr>
<td style="text-align:left;">
Npbwr1|Npbwr1
</td>
<td style="text-align:left;">
635.79705
</td>
<td style="text-align:left;">
3.8208865
</td>
<td style="text-align:left;">
0.7024514
</td>
<td style="text-align:left;">
5.4393604
</td>
<td style="text-align:left;">
0.0000001
</td>
<td style="text-align:left;">
0.0000017
</td>
</tr>
<tr>
<td style="text-align:left;">
Oprk1|Oprk1
</td>
<td style="text-align:left;">
8636.39912
</td>
<td style="text-align:left;">
3.4664690
</td>
<td style="text-align:left;">
0.3276905
</td>
<td style="text-align:left;">
10.5784841
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
St18|St18
</td>
<td style="text-align:left;">
1253.13436
</td>
<td style="text-align:left;">
-1.8851226
</td>
<td style="text-align:left;">
0.5860577
</td>
<td style="text-align:left;">
-3.2166162
</td>
<td style="text-align:left;">
0.0012971
</td>
<td style="text-align:left;">
0.0099514
</td>
</tr>
<tr>
<td style="text-align:left;">
Rb1cc1|Rb1cc1
</td>
<td style="text-align:left;">
10439.08996
</td>
<td style="text-align:left;">
0.4578156
</td>
<td style="text-align:left;">
0.2934158
</td>
<td style="text-align:left;">
1.5602964
</td>
<td style="text-align:left;">
0.1186899
</td>
<td style="text-align:left;">
0.2966976
</td>
</tr>
<tr>
<td style="text-align:left;">
4732440D04Rik|4732440D04Rik
</td>
<td style="text-align:left;">
4779.55239
</td>
<td style="text-align:left;">
1.0960037
</td>
<td style="text-align:left;">
0.3040615
</td>
<td style="text-align:left;">
3.6045467
</td>
<td style="text-align:left;">
0.0003127
</td>
<td style="text-align:left;">
0.0030441
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm26901|Gm26901
</td>
<td style="text-align:left;">
206.26282
</td>
<td style="text-align:left;">
0.0161455
</td>
<td style="text-align:left;">
0.9519191
</td>
<td style="text-align:left;">
0.0169610
</td>
<td style="text-align:left;">
0.9864677
</td>
<td style="text-align:left;">
0.9949735
</td>
</tr>
<tr>
<td style="text-align:left;">
Pcmtd1|Pcmtd1
</td>
<td style="text-align:left;">
4018.40324
</td>
<td style="text-align:left;">
-0.1887026
</td>
<td style="text-align:left;">
0.3090352
</td>
<td style="text-align:left;">
-0.6106184
</td>
<td style="text-align:left;">
0.5414523
</td>
<td style="text-align:left;">
0.7463224
</td>
</tr>
<tr>
<td style="text-align:left;">
Rrs1|Rrs1
</td>
<td style="text-align:left;">
735.37173
</td>
<td style="text-align:left;">
0.0650079
</td>
<td style="text-align:left;">
0.3863280
</td>
<td style="text-align:left;">
0.1682714
</td>
<td style="text-align:left;">
0.8663698
</td>
<td style="text-align:left;">
0.9433220
</td>
</tr>
<tr>
<td style="text-align:left;">
Mybl1|Mybl1
</td>
<td style="text-align:left;">
138.64094
</td>
<td style="text-align:left;">
-0.1521019
</td>
<td style="text-align:left;">
0.8982176
</td>
<td style="text-align:left;">
-0.1693374
</td>
<td style="text-align:left;">
0.8655312
</td>
<td style="text-align:left;">
0.9432810
</td>
</tr>
<tr>
<td style="text-align:left;">
Adhfe1|Adhfe1
</td>
<td style="text-align:left;">
709.72361
</td>
<td style="text-align:left;">
-0.5002650
</td>
<td style="text-align:left;">
0.6650929
</td>
<td style="text-align:left;">
-0.7521731
</td>
<td style="text-align:left;">
0.4519470
</td>
<td style="text-align:left;">
0.6739242
</td>
</tr>
</tbody>
</table>
</div>
<p>out of 26336 with nonzero total read count adjusted p-value &lt; 0.1
LFC &gt; 0 (up) : 1934, 7.3% LFC &lt; 0 (down) : 3517, 13% outliers [1]
: 908, 3.4% low counts [2] : 3505, 13% (mean count &lt; 3) [1] see
‘cooksCutoff’ argument of ?results [2] see ‘independentFiltering’
argument of ?results ## Annotate Results Files and Pull Significant DEGs
{.tabset}</p>
<p>This subsection is for annotation the results files to include all
necessary information for downstream analyses.</p>
<p>After annotation I subset the results to only view the significantly
differentially expressed genes with an adjusted p-value &gt; 0.05.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Subset for significant genes only</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a>results_sig <span class="ot">&lt;-</span> <span class="fu">subset</span>(res, padj <span class="sc">&lt;</span> <span class="fl">0.05</span>)</span></code></pre></div>
</div>
<div id="da-vs-nda" class="section level3" number="2.1.2">
<h3><span class="header-section-number">2.1.2</span> DA vs nDA</h3>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:1000px; ">
<table class="table table-striped table-hover table-condensed table-responsive" style="font-size: 10px; font-family: Arial; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Significant DEGs DA vs nDA
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
baseMean
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
log2FoldChange
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
lfcSE
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
stat
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
padj
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
symbol
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
entrez
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ensembl
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Gm39585|Gm39585
</td>
<td style="text-align:left;">
7249.50804
</td>
<td style="text-align:left;">
-1.250087
</td>
<td style="text-align:left;">
0.3299238
</td>
<td style="text-align:left;">
-3.789017
</td>
<td style="text-align:left;">
0.0001512
</td>
<td style="text-align:left;">
0.0016620
</td>
<td style="text-align:left;">
Gm39585
</td>
<td style="text-align:left;">
predicted gene, 39585
</td>
<td style="text-align:left;">
105243853
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm16041|Gm16041
</td>
<td style="text-align:left;">
54.96913
</td>
<td style="text-align:left;">
-9.213591
</td>
<td style="text-align:left;">
1.5556113
</td>
<td style="text-align:left;">
-5.922811
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000001
</td>
<td style="text-align:left;">
Gm16041
</td>
<td style="text-align:left;">
predicted gene 16041
</td>
<td style="text-align:left;">
102631647
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Atp6v1h|Atp6v1h
</td>
<td style="text-align:left;">
8199.89727
</td>
<td style="text-align:left;">
1.532529
</td>
<td style="text-align:left;">
0.1984866
</td>
<td style="text-align:left;">
7.721072
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
Atp6v1h
</td>
<td style="text-align:left;">
ATPase, H+ transporting, lysosomal V1 subunit H
</td>
<td style="text-align:left;">
108664
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Npbwr1|Npbwr1
</td>
<td style="text-align:left;">
635.79705
</td>
<td style="text-align:left;">
3.820886
</td>
<td style="text-align:left;">
0.7024514
</td>
<td style="text-align:left;">
5.439360
</td>
<td style="text-align:left;">
0.0000001
</td>
<td style="text-align:left;">
0.0000017
</td>
<td style="text-align:left;">
Npbwr1
</td>
<td style="text-align:left;">
neuropeptides B/W receptor 1
</td>
<td style="text-align:left;">
226304
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Oprk1|Oprk1
</td>
<td style="text-align:left;">
8636.39912
</td>
<td style="text-align:left;">
3.466469
</td>
<td style="text-align:left;">
0.3276905
</td>
<td style="text-align:left;">
10.578484
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
Oprk1
</td>
<td style="text-align:left;">
opioid receptor, kappa 1
</td>
<td style="text-align:left;">
18387
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
St18|St18
</td>
<td style="text-align:left;">
1253.13436
</td>
<td style="text-align:left;">
-1.885123
</td>
<td style="text-align:left;">
0.5860577
</td>
<td style="text-align:left;">
-3.216616
</td>
<td style="text-align:left;">
0.0012971
</td>
<td style="text-align:left;">
0.0099514
</td>
<td style="text-align:left;">
St18
</td>
<td style="text-align:left;">
suppression of tumorigenicity 18
</td>
<td style="text-align:left;">
240690
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
4732440D04Rik|4732440D04Rik
</td>
<td style="text-align:left;">
4779.55239
</td>
<td style="text-align:left;">
1.096004
</td>
<td style="text-align:left;">
0.3040615
</td>
<td style="text-align:left;">
3.604547
</td>
<td style="text-align:left;">
0.0003127
</td>
<td style="text-align:left;">
0.0030441
</td>
<td style="text-align:left;">
4732440D04Rik
</td>
<td style="text-align:left;">
RIKEN cDNA 4732440D04 gene
</td>
<td style="text-align:left;">
654788
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
Sntg1|Sntg1
</td>
<td style="text-align:left;">
2214.91833
</td>
<td style="text-align:left;">
1.178273
</td>
<td style="text-align:left;">
0.4156389
</td>
<td style="text-align:left;">
2.834849
</td>
<td style="text-align:left;">
0.0045847
</td>
<td style="text-align:left;">
0.0273723
</td>
<td style="text-align:left;">
Sntg1
</td>
<td style="text-align:left;">
syntrophin, gamma 1
</td>
<td style="text-align:left;">
71096
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Sgk3|Sgk3
</td>
<td style="text-align:left;">
797.14010
</td>
<td style="text-align:left;">
-2.172757
</td>
<td style="text-align:left;">
0.4461371
</td>
<td style="text-align:left;">
-4.870157
</td>
<td style="text-align:left;">
0.0000011
</td>
<td style="text-align:left;">
0.0000241
</td>
<td style="text-align:left;">
Sgk3
</td>
<td style="text-align:left;">
serum/glucocorticoid regulated kinase 3
</td>
<td style="text-align:left;">
170755
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Vcpip1|Vcpip1
</td>
<td style="text-align:left;">
5991.43803
</td>
<td style="text-align:left;">
0.917267
</td>
<td style="text-align:left;">
0.2377693
</td>
<td style="text-align:left;">
3.857802
</td>
<td style="text-align:left;">
0.0001144
</td>
<td style="text-align:left;">
0.0013112
</td>
<td style="text-align:left;">
Vcpip1
</td>
<td style="text-align:left;">
valosin containing protein (p97)/p47 complex interacting protein 1
</td>
<td style="text-align:left;">
70675
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Cops5|Cops5
</td>
<td style="text-align:left;">
2116.70793
</td>
<td style="text-align:left;">
1.060638
</td>
<td style="text-align:left;">
0.2069314
</td>
<td style="text-align:left;">
5.125554
</td>
<td style="text-align:left;">
0.0000003
</td>
<td style="text-align:left;">
0.0000075
</td>
<td style="text-align:left;">
Cops5
</td>
<td style="text-align:left;">
COP9 signalosome subunit 5
</td>
<td style="text-align:left;">
26754
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Lactb2|Lactb2
</td>
<td style="text-align:left;">
1616.47178
</td>
<td style="text-align:left;">
1.435364
</td>
<td style="text-align:left;">
0.5337942
</td>
<td style="text-align:left;">
2.688983
</td>
<td style="text-align:left;">
0.0071670
</td>
<td style="text-align:left;">
0.0386536
</td>
<td style="text-align:left;">
Lactb2
</td>
<td style="text-align:left;">
lactamase, beta 2
</td>
<td style="text-align:left;">
212442
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
LOC102634042|LOC102634042
</td>
<td style="text-align:left;">
34.38388
</td>
<td style="text-align:left;">
-8.536588
</td>
<td style="text-align:left;">
1.7392784
</td>
<td style="text-align:left;">
-4.908120
</td>
<td style="text-align:left;">
0.0000009
</td>
<td style="text-align:left;">
0.0000203
</td>
<td style="text-align:left;">
LOC102634042
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Tfap2b|Tfap2b
</td>
<td style="text-align:left;">
34.30287
</td>
<td style="text-align:left;">
-8.533309
</td>
<td style="text-align:left;">
2.0629577
</td>
<td style="text-align:left;">
-4.136444
</td>
<td style="text-align:left;">
0.0000353
</td>
<td style="text-align:left;">
0.0004876
</td>
<td style="text-align:left;">
Tfap2b
</td>
<td style="text-align:left;">
transcription factor AP-2 beta
</td>
<td style="text-align:left;">
21419
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm28836|Gm28836
</td>
<td style="text-align:left;">
13.26756
</td>
<td style="text-align:left;">
-7.162762
</td>
<td style="text-align:left;">
2.3185615
</td>
<td style="text-align:left;">
-3.089313
</td>
<td style="text-align:left;">
0.0020062
</td>
<td style="text-align:left;">
0.0141293
</td>
<td style="text-align:left;">
Gm28836
</td>
<td style="text-align:left;">
predicted gene 28836
</td>
<td style="text-align:left;">
102635593
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
Col19a1|Col19a1
</td>
<td style="text-align:left;">
1842.71516
</td>
<td style="text-align:left;">
-1.520178
</td>
<td style="text-align:left;">
0.5653375
</td>
<td style="text-align:left;">
-2.688974
</td>
<td style="text-align:left;">
0.0071672
</td>
<td style="text-align:left;">
0.0386536
</td>
<td style="text-align:left;">
Col19a1
</td>
<td style="text-align:left;">
collagen, type XIX, alpha 1
</td>
<td style="text-align:left;">
12823
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm20172|Gm20172
</td>
<td style="text-align:left;">
207.52086
</td>
<td style="text-align:left;">
-2.996416
</td>
<td style="text-align:left;">
0.8784245
</td>
<td style="text-align:left;">
-3.411125
</td>
<td style="text-align:left;">
0.0006470
</td>
<td style="text-align:left;">
0.0056238
</td>
<td style="text-align:left;">
Gm20172
</td>
<td style="text-align:left;">
predicted gene, 20172
</td>
<td style="text-align:left;">
100504323
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm9839|Gm9839
</td>
<td style="text-align:left;">
21.91732
</td>
<td style="text-align:left;">
-7.886899
</td>
<td style="text-align:left;">
2.0845273
</td>
<td style="text-align:left;">
-3.783543
</td>
<td style="text-align:left;">
0.0001546
</td>
<td style="text-align:left;">
0.0016889
</td>
<td style="text-align:left;">
Gm9839
</td>
<td style="text-align:left;">
predicted gene 9839
</td>
<td style="text-align:left;">
408192
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm32790|Gm32790
</td>
<td style="text-align:left;">
23.58636
</td>
<td style="text-align:left;">
-7.993022
</td>
<td style="text-align:left;">
1.9845776
</td>
<td style="text-align:left;">
-4.027569
</td>
<td style="text-align:left;">
0.0000564
</td>
<td style="text-align:left;">
0.0007246
</td>
<td style="text-align:left;">
Gm32790
</td>
<td style="text-align:left;">
predicted gene, 32790
</td>
<td style="text-align:left;">
102635458
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
Dst|Dst
</td>
<td style="text-align:left;">
25341.58858
</td>
<td style="text-align:left;">
-1.009630
</td>
<td style="text-align:left;">
0.1547716
</td>
<td style="text-align:left;">
-6.523356
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
Dst
</td>
<td style="text-align:left;">
dystonin
</td>
<td style="text-align:left;">
13518
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="visualize-shrinkage-estimations" class="section level2"
number="2.2">
<h2><span class="header-section-number">2.2</span> Visualize Shrinkage
Estimations</h2>
<p>Following subsetting, I want to see how the data looks based on
effect size so I have run a shrinkage model using the “apeglm” method
which normalizes counts based on effect size without major changes to
data structure. I have included non-shrunk models as well for
comparison.</p>
<div id="non-shrunken-l2fc" class="section level3 tabset"
number="2.2.1">
<h3 class="tabset"><span class="header-section-number">2.2.1</span>
Non-Shrunken L2FC</h3>
<div id="da-vs-nda-1" class="section level4" number="2.2.1.1">
<h4><span class="header-section-number">2.2.1.1</span> DA vs nDA</h4>
<p><img src="figure/DAvsnDA.Rmd/Non-Shrunken%20Estimate%20Plots-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Non-Shrunken-Estimate-Plots-1">
Past versions of "Non-Shrunken Estimate Plots-1.png"
</button>
</p>
<div id="fig-no-spaces-Non-Shrunken-Estimate-Plots-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/1ee1d086cd39bf3115343f820fe5f9cd5de1d6a1/docs/figure/DAvsnDA.Rmd/Non-Shrunken Estimate Plots-1.png" target="_blank">1ee1d08</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="shrunken-l2fc" class="section level3 tabset" number="2.2.2">
<h3 class="tabset"><span class="header-section-number">2.2.2</span>
Shrunken L2FC</h3>
<div id="da-vs-nda-2" class="section level4" number="2.2.2.1">
<h4><span class="header-section-number">2.2.2.1</span> DA vs nDA</h4>
<p><img src="figure/DAvsnDA.Rmd/Shrunken%20Estimate%20Plots-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Shrunken-Estimate-Plots-1">
Past versions of "Shrunken Estimate Plots-1.png"
</button>
</p>
<div id="fig-no-spaces-Shrunken-Estimate-Plots-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/1ee1d086cd39bf3115343f820fe5f9cd5de1d6a1/docs/figure/DAvsnDA.Rmd/Shrunken Estimate Plots-1.png" target="_blank">1ee1d08</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
</div>
<div id="looking-at-differentially-expressed-genes"
class="section level1" number="3">
<h1><span class="header-section-number">3</span> Looking at
Differentially Expressed Genes</h1>
<p>We can generate a matrix to identify where our differentially
expressed genes lie between each group based on an alpha value of
0.05.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Matrix Visualizaattion</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>x <span class="ot">&lt;-</span> <span class="fu">vsDEGMatrix</span>(</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">data =</span> dds, <span class="at">padj =</span> <span class="fl">0.05</span>, <span class="at">d.factor =</span> <span class="st">&quot;condition&quot;</span>,</span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;deseq&quot;</span>, <span class="at">title =</span> <span class="cn">TRUE</span>, <span class="at">legend =</span> <span class="cn">TRUE</span>, <span class="at">grid =</span> <span class="cn">TRUE</span></span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a>)</span></code></pre></div>
<p><img src="figure/DAvsnDA.Rmd/Matrix%20of%20Differentially%20Expressed%20genes%20across%20groups-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Matrix-of-Differentially-Expressed-genes-across-groups-1">
Past versions of "Matrix of Differentially Expressed genes across
groups-1.png"
</button>
</p>
<div
id="fig-no-spaces-Matrix-of-Differentially-Expressed-genes-across-groups-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/1ee1d086cd39bf3115343f820fe5f9cd5de1d6a1/docs/figure/DAvsnDA.Rmd/Matrix of Differentially Expressed genes across groups-1.png" target="_blank">1ee1d08</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="data-visualization" class="section level1" number="4">
<h1><span class="header-section-number">4</span> Data Visualization</h1>
<div id="normalized-counts-and-plots-of-genes-of-interest"
class="section level2 tabset" number="4.1">
<h2 class="tabset"><span class="header-section-number">4.1</span>
Normalized Counts and Plots of Genes of Interest</h2>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>level_order <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">&quot;DA&quot;</span>, <span class="st">&quot;nDA&quot;</span>) <span class="co"># this vector might be useful for other plots/analyses</span></span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a>normalized_counts <span class="ot">&lt;-</span> <span class="fu">counts</span>(dds, <span class="at">normalized =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
<div id="th" class="section level3" number="4.1.1">
<h3><span class="header-section-number">4.1.1</span> Th</h3>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="do">#### Th</span></span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>d2 <span class="ot">&lt;-</span> <span class="fu">plotCounts</span>(dds,</span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">gene =</span> <span class="st">&quot;Th|Th&quot;</span>, <span class="at">intgroup =</span> <span class="st">&quot;condition&quot;</span>,</span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">returnData =</span> <span class="cn">TRUE</span></span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a>a <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(d2, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">factor</span>(condition, <span class="at">level =</span> level_order), <span class="at">y =</span> count, <span class="at">color =</span> condition)) <span class="sc">+</span></span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">position =</span> <span class="fu">position_jitter</span>(<span class="at">w =</span> <span class="fl">0.1</span>, <span class="at">h =</span> <span class="dv">0</span>), <span class="at">size =</span> <span class="dv">5</span>) <span class="sc">+</span></span>
<span id="cb4-10"><a href="#cb4-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values=</span><span class="fu">c</span>(<span class="st">&quot;#FF9289&quot;</span>, <span class="st">&quot;#00DAE0&quot;</span>)) <span class="sc">+</span></span>
<span id="cb4-11"><a href="#cb4-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb4-12"><a href="#cb4-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Condition&quot;</span>)<span class="sc">+</span></span>
<span id="cb4-13"><a href="#cb4-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Normalized Counts&quot;</span>)<span class="sc">+</span></span>
<span id="cb4-14"><a href="#cb4-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggtitle</span>(</span>
<span id="cb4-15"><a href="#cb4-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">label =</span> <span class="st">&quot;Th Expression&quot;</span>,</span>
<span id="cb4-16"><a href="#cb4-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">subtitle =</span> <span class="st">&quot;Normalized Gene Expression&quot;</span></span>
<span id="cb4-17"><a href="#cb4-17" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb4-18"><a href="#cb4-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-19"><a href="#cb4-19" aria-hidden="true" tabindex="-1"></a>a</span></code></pre></div>
<p><img src="figure/DAvsnDA.Rmd/Selected%20Genes-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Selected-Genes-1">
Past versions of "Selected Genes-1.png"
</button>
</p>
<div id="fig-no-spaces-Selected-Genes-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/7c1df5e6061f4671761a1740a2132edfc3958596/docs/figure/DAvsnDA.Rmd/Selected Genes-1.png" target="_blank">7c1df5e</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/c31538bbb33383ec16632a62b439809eb31d4b2e/docs/figure/DAvsnDA.Rmd/Selected Genes-1.png" target="_blank">c31538b</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/bfc97c14694a0cfb3bb5699555e3e157ce3f3035/docs/figure/DAvsnDA.Rmd/Selected Genes-1.png" target="_blank">bfc97c1</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/1ee1d086cd39bf3115343f820fe5f9cd5de1d6a1/docs/figure/DAvsnDA.Rmd/Selected Genes-1.png" target="_blank">1ee1d08</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="en1" class="section level3" number="4.1.2">
<h3><span class="header-section-number">4.1.2</span> En1</h3>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="do">#### En1</span></span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a>d2 <span class="ot">&lt;-</span> <span class="fu">plotCounts</span>(dds,</span>
<span id="cb5-5"><a href="#cb5-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">gene =</span> <span class="st">&quot;En1|En1&quot;</span>, <span class="at">intgroup =</span> <span class="st">&quot;condition&quot;</span>,</span>
<span id="cb5-6"><a href="#cb5-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">returnData =</span> <span class="cn">TRUE</span></span>
<span id="cb5-7"><a href="#cb5-7" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb5-8"><a href="#cb5-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-9"><a href="#cb5-9" aria-hidden="true" tabindex="-1"></a>a <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(d2, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">factor</span>(condition, <span class="at">level =</span> level_order), <span class="at">y =</span> count, <span class="at">color =</span> condition)) <span class="sc">+</span></span>
<span id="cb5-10"><a href="#cb5-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">position =</span> <span class="fu">position_jitter</span>(<span class="at">w =</span> <span class="fl">0.1</span>, <span class="at">h =</span> <span class="dv">0</span>), <span class="at">size =</span> <span class="dv">5</span>) <span class="sc">+</span></span>
<span id="cb5-11"><a href="#cb5-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values=</span><span class="fu">c</span>(<span class="st">&quot;#FF9289&quot;</span>, <span class="st">&quot;#00DAE0&quot;</span>)) <span class="sc">+</span></span>
<span id="cb5-12"><a href="#cb5-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb5-13"><a href="#cb5-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Condition&quot;</span>)<span class="sc">+</span></span>
<span id="cb5-14"><a href="#cb5-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Normalized Counts&quot;</span>)<span class="sc">+</span></span>
<span id="cb5-15"><a href="#cb5-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggtitle</span>(</span>
<span id="cb5-16"><a href="#cb5-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">label =</span> <span class="st">&quot;En1 Expression&quot;</span>,</span>
<span id="cb5-17"><a href="#cb5-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">subtitle =</span> <span class="st">&quot;Normalized Gene Expression&quot;</span></span>
<span id="cb5-18"><a href="#cb5-18" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb5-19"><a href="#cb5-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-20"><a href="#cb5-20" aria-hidden="true" tabindex="-1"></a>a</span></code></pre></div>
<p><img src="figure/DAvsnDA.Rmd/Selected%20Genes2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Selected-Genes2-1">
Past versions of "Selected Genes2-1.png"
</button>
</p>
<div id="fig-no-spaces-Selected-Genes2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/7c1df5e6061f4671761a1740a2132edfc3958596/docs/figure/DAvsnDA.Rmd/Selected Genes2-1.png" target="_blank">7c1df5e</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/c31538bbb33383ec16632a62b439809eb31d4b2e/docs/figure/DAvsnDA.Rmd/Selected Genes2-1.png" target="_blank">c31538b</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/bfc97c14694a0cfb3bb5699555e3e157ce3f3035/docs/figure/DAvsnDA.Rmd/Selected Genes2-1.png" target="_blank">bfc97c1</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/1ee1d086cd39bf3115343f820fe5f9cd5de1d6a1/docs/figure/DAvsnDA.Rmd/Selected Genes2-1.png" target="_blank">1ee1d08</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="slc6a3" class="section level3" number="4.1.3">
<h3><span class="header-section-number">4.1.3</span> Slc6a3</h3>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="do">#### Slc6a3</span></span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a>d2 <span class="ot">&lt;-</span> <span class="fu">plotCounts</span>(dds,</span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">gene =</span> <span class="st">&quot;Slc6a3|Slc6a3&quot;</span>, <span class="at">intgroup =</span> <span class="st">&quot;condition&quot;</span>,</span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">returnData =</span> <span class="cn">TRUE</span></span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb6-8"><a href="#cb6-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-9"><a href="#cb6-9" aria-hidden="true" tabindex="-1"></a>a <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(d2, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">factor</span>(condition, <span class="at">level =</span> level_order), <span class="at">y =</span> count, <span class="at">color =</span> condition)) <span class="sc">+</span></span>
<span id="cb6-10"><a href="#cb6-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">position =</span> <span class="fu">position_jitter</span>(<span class="at">w =</span> <span class="fl">0.1</span>, <span class="at">h =</span> <span class="dv">0</span>), <span class="at">size =</span> <span class="dv">5</span>) <span class="sc">+</span></span>
<span id="cb6-11"><a href="#cb6-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values=</span><span class="fu">c</span>(<span class="st">&quot;#FF9289&quot;</span>, <span class="st">&quot;#00DAE0&quot;</span>)) <span class="sc">+</span></span>
<span id="cb6-12"><a href="#cb6-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb6-13"><a href="#cb6-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Condition&quot;</span>)<span class="sc">+</span></span>
<span id="cb6-14"><a href="#cb6-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Normalized Counts&quot;</span>)<span class="sc">+</span></span>
<span id="cb6-15"><a href="#cb6-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggtitle</span>(</span>
<span id="cb6-16"><a href="#cb6-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">label =</span> <span class="st">&quot;Slc6a3 Expression&quot;</span>,</span>
<span id="cb6-17"><a href="#cb6-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">subtitle =</span> <span class="st">&quot;Normalized Gene Expression&quot;</span></span>
<span id="cb6-18"><a href="#cb6-18" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb6-19"><a href="#cb6-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-20"><a href="#cb6-20" aria-hidden="true" tabindex="-1"></a>a</span></code></pre></div>
<p><img src="figure/DAvsnDA.Rmd/Selected%20Genes3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Selected-Genes3-1">
Past versions of "Selected Genes3-1.png"
</button>
</p>
<div id="fig-no-spaces-Selected-Genes3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/7c1df5e6061f4671761a1740a2132edfc3958596/docs/figure/DAvsnDA.Rmd/Selected Genes3-1.png" target="_blank">7c1df5e</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/c31538bbb33383ec16632a62b439809eb31d4b2e/docs/figure/DAvsnDA.Rmd/Selected Genes3-1.png" target="_blank">c31538b</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/bfc97c14694a0cfb3bb5699555e3e157ce3f3035/docs/figure/DAvsnDA.Rmd/Selected Genes3-1.png" target="_blank">bfc97c1</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/1ee1d086cd39bf3115343f820fe5f9cd5de1d6a1/docs/figure/DAvsnDA.Rmd/Selected Genes3-1.png" target="_blank">1ee1d08</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="neat1" class="section level3" number="4.1.4">
<h3><span class="header-section-number">4.1.4</span> Neat1</h3>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="do">#### Ehmt2</span></span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a>d2 <span class="ot">&lt;-</span> <span class="fu">plotCounts</span>(dds,</span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">gene =</span> <span class="st">&quot;Neat1|Neat1&quot;</span>, <span class="at">intgroup =</span> <span class="st">&quot;condition&quot;</span>,</span>
<span id="cb7-5"><a href="#cb7-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">returnData =</span> <span class="cn">TRUE</span></span>
<span id="cb7-6"><a href="#cb7-6" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb7-7"><a href="#cb7-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-8"><a href="#cb7-8" aria-hidden="true" tabindex="-1"></a>a <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(d2, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">factor</span>(condition, <span class="at">level =</span> level_order), <span class="at">y =</span> count, <span class="at">color =</span> condition)) <span class="sc">+</span></span>
<span id="cb7-9"><a href="#cb7-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">position =</span> <span class="fu">position_jitter</span>(<span class="at">w =</span> <span class="fl">0.1</span>, <span class="at">h =</span> <span class="dv">0</span>), <span class="at">size =</span> <span class="dv">5</span>) <span class="sc">+</span></span>
<span id="cb7-10"><a href="#cb7-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values=</span><span class="fu">c</span>(<span class="st">&quot;#FF9289&quot;</span>, <span class="st">&quot;#00DAE0&quot;</span>)) <span class="sc">+</span></span>
<span id="cb7-11"><a href="#cb7-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb7-12"><a href="#cb7-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Condition&quot;</span>)<span class="sc">+</span></span>
<span id="cb7-13"><a href="#cb7-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Normalized Counts&quot;</span>)<span class="sc">+</span></span>
<span id="cb7-14"><a href="#cb7-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggtitle</span>(</span>
<span id="cb7-15"><a href="#cb7-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">label =</span> <span class="st">&quot;Neat1 Expression&quot;</span>,</span>
<span id="cb7-16"><a href="#cb7-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">subtitle =</span> <span class="st">&quot;Normalized Gene Expression&quot;</span></span>
<span id="cb7-17"><a href="#cb7-17" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb7-18"><a href="#cb7-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-19"><a href="#cb7-19" aria-hidden="true" tabindex="-1"></a>a</span></code></pre></div>
<p><img src="figure/DAvsnDA.Rmd/Selected%20Genes4-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Selected-Genes4-1">
Past versions of "Selected Genes4-1.png"
</button>
</p>
<div id="fig-no-spaces-Selected-Genes4-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/7c1df5e6061f4671761a1740a2132edfc3958596/docs/figure/DAvsnDA.Rmd/Selected Genes4-1.png" target="_blank">7c1df5e</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/c31538bbb33383ec16632a62b439809eb31d4b2e/docs/figure/DAvsnDA.Rmd/Selected Genes4-1.png" target="_blank">c31538b</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/bfc97c14694a0cfb3bb5699555e3e157ce3f3035/docs/figure/DAvsnDA.Rmd/Selected Genes4-1.png" target="_blank">bfc97c1</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/1ee1d086cd39bf3115343f820fe5f9cd5de1d6a1/docs/figure/DAvsnDA.Rmd/Selected Genes4-1.png" target="_blank">1ee1d08</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="mag" class="section level3" number="4.1.5">
<h3><span class="header-section-number">4.1.5</span> Mag</h3>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="do">#### Mag</span></span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-3"><a href="#cb8-3" aria-hidden="true" tabindex="-1"></a>d2 <span class="ot">&lt;-</span> <span class="fu">plotCounts</span>(dds,</span>
<span id="cb8-4"><a href="#cb8-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">gene =</span> <span class="st">&quot;Mag|Mag&quot;</span>, <span class="at">intgroup =</span> <span class="st">&quot;condition&quot;</span>,</span>
<span id="cb8-5"><a href="#cb8-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">returnData =</span> <span class="cn">TRUE</span></span>
<span id="cb8-6"><a href="#cb8-6" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb8-7"><a href="#cb8-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-8"><a href="#cb8-8" aria-hidden="true" tabindex="-1"></a>a <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(d2, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">factor</span>(condition, <span class="at">level =</span> level_order), <span class="at">y =</span> count, <span class="at">color =</span> condition)) <span class="sc">+</span></span>
<span id="cb8-9"><a href="#cb8-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">position =</span> <span class="fu">position_jitter</span>(<span class="at">w =</span> <span class="fl">0.1</span>, <span class="at">h =</span> <span class="dv">0</span>), <span class="at">size =</span> <span class="dv">5</span>) <span class="sc">+</span></span>
<span id="cb8-10"><a href="#cb8-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values=</span><span class="fu">c</span>(<span class="st">&quot;#FF9289&quot;</span>, <span class="st">&quot;#00DAE0&quot;</span>)) <span class="sc">+</span></span>
<span id="cb8-11"><a href="#cb8-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb8-12"><a href="#cb8-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Condition&quot;</span>)<span class="sc">+</span></span>
<span id="cb8-13"><a href="#cb8-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Normalized Counts&quot;</span>)<span class="sc">+</span></span>
<span id="cb8-14"><a href="#cb8-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggtitle</span>(</span>
<span id="cb8-15"><a href="#cb8-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">label =</span> <span class="st">&quot;Mag Expression&quot;</span>,</span>
<span id="cb8-16"><a href="#cb8-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">subtitle =</span> <span class="st">&quot;Normalized Gene Expression&quot;</span></span>
<span id="cb8-17"><a href="#cb8-17" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb8-18"><a href="#cb8-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-19"><a href="#cb8-19" aria-hidden="true" tabindex="-1"></a>a</span></code></pre></div>
<p><img src="figure/DAvsnDA.Rmd/Selected%20Genes5-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Selected-Genes5-1">
Past versions of "Selected Genes5-1.png"
</button>
</p>
<div id="fig-no-spaces-Selected-Genes5-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/7c1df5e6061f4671761a1740a2132edfc3958596/docs/figure/DAvsnDA.Rmd/Selected Genes5-1.png" target="_blank">7c1df5e</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/c31538bbb33383ec16632a62b439809eb31d4b2e/docs/figure/DAvsnDA.Rmd/Selected Genes5-1.png" target="_blank">c31538b</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/bfc97c14694a0cfb3bb5699555e3e157ce3f3035/docs/figure/DAvsnDA.Rmd/Selected Genes5-1.png" target="_blank">bfc97c1</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/1ee1d086cd39bf3115343f820fe5f9cd5de1d6a1/docs/figure/DAvsnDA.Rmd/Selected Genes5-1.png" target="_blank">1ee1d08</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="principal-components-analysis-pca" class="section level2"
number="4.2">
<h2><span class="header-section-number">4.2</span> Principal Components
Analysis (PCA)</h2>
<p>For visualization I am converting all analyses to logarithmic.
Following this I have completed a PCA.</p>
<p><img src="figure/DAvsnDA.Rmd/Plot%20PCA-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Plot-PCA-1">
Past versions of "Plot PCA-1.png"
</button>
</p>
<div id="fig-no-spaces-Plot-PCA-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/7c1df5e6061f4671761a1740a2132edfc3958596/docs/figure/DAvsnDA.Rmd/Plot PCA-1.png" target="_blank">7c1df5e</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/c31538bbb33383ec16632a62b439809eb31d4b2e/docs/figure/DAvsnDA.Rmd/Plot PCA-1.png" target="_blank">c31538b</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/bfc97c14694a0cfb3bb5699555e3e157ce3f3035/docs/figure/DAvsnDA.Rmd/Plot PCA-1.png" target="_blank">bfc97c1</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/1ee1d086cd39bf3115343f820fe5f9cd5de1d6a1/docs/figure/DAvsnDA.Rmd/Plot PCA-1.png" target="_blank">1ee1d08</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a>ressig1 <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(results_sig)</span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a>ressig1<span class="sc">$</span>Comparison <span class="ot">&lt;-</span> <span class="st">&quot;DA vs nDA&quot;</span></span>
<span id="cb9-4"><a href="#cb9-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-5"><a href="#cb9-5" aria-hidden="true" tabindex="-1"></a>ressig1_highlfc_pos <span class="ot">&lt;-</span> ressig1[ressig1<span class="sc">$</span>log2FoldChange <span class="sc">&gt;=</span> <span class="fl">1.5</span>,]</span>
<span id="cb9-6"><a href="#cb9-6" aria-hidden="true" tabindex="-1"></a>ressig1_highlfc_neg <span class="ot">&lt;-</span> ressig1[ressig1<span class="sc">$</span>log2FoldChange <span class="sc">&lt;=</span> <span class="sc">-</span><span class="fl">1.5</span>,]</span>
<span id="cb9-7"><a href="#cb9-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-8"><a href="#cb9-8" aria-hidden="true" tabindex="-1"></a>temp_sigdegshm <span class="ot">&lt;-</span> <span class="fu">rbind</span>(ressig1_highlfc_pos, ressig1_highlfc_neg)</span>
<span id="cb9-9"><a href="#cb9-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-10"><a href="#cb9-10" aria-hidden="true" tabindex="-1"></a>temp_sigdegshm6 <span class="ot">&lt;-</span> temp_sigdegshm[temp_sigdegshm<span class="sc">$</span>lfcSE <span class="sc">&lt;=</span><span class="dv">1</span>,]</span>
<span id="cb9-11"><a href="#cb9-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-12"><a href="#cb9-12" aria-hidden="true" tabindex="-1"></a>temp_sigdegshm6 <span class="ot">&lt;-</span> temp_sigdegshm6[<span class="fu">order</span>(temp_sigdegshm6<span class="sc">$</span>log2FoldChange),]</span>
<span id="cb9-13"><a href="#cb9-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-14"><a href="#cb9-14" aria-hidden="true" tabindex="-1"></a><span class="co">#sum(is.na(temp_sigdegshm6$symbol))</span></span>
<span id="cb9-15"><a href="#cb9-15" aria-hidden="true" tabindex="-1"></a><span class="co">#sum(duplicated(temp_sigdegshm6$symbol))</span></span>
<span id="cb9-16"><a href="#cb9-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-17"><a href="#cb9-17" aria-hidden="true" tabindex="-1"></a>dup_rows <span class="ot">&lt;-</span> <span class="fu">duplicated</span>(temp_sigdegshm6<span class="sc">$</span>symbol)</span>
<span id="cb9-18"><a href="#cb9-18" aria-hidden="true" tabindex="-1"></a>data_dedup <span class="ot">&lt;-</span> temp_sigdegshm6[<span class="sc">!</span>dup_rows, ]</span>
<span id="cb9-19"><a href="#cb9-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-20"><a href="#cb9-20" aria-hidden="true" tabindex="-1"></a>rows <span class="ot">&lt;-</span> <span class="fu">rownames</span>(data_dedup)</span>
<span id="cb9-21"><a href="#cb9-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-22"><a href="#cb9-22" aria-hidden="true" tabindex="-1"></a>select <span class="ot">&lt;-</span> <span class="fu">order</span>(<span class="fu">rowMeans</span>(<span class="fu">counts</span>(dds,<span class="at">normalized=</span><span class="cn">TRUE</span>)),</span>
<span id="cb9-23"><a href="#cb9-23" aria-hidden="true" tabindex="-1"></a>                <span class="at">decreasing=</span><span class="cn">TRUE</span>)<span class="co">#[1:1500]</span></span>
<span id="cb9-24"><a href="#cb9-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-25"><a href="#cb9-25" aria-hidden="true" tabindex="-1"></a>mat <span class="ot">&lt;-</span> <span class="fu">assay</span>(ddsMat_rlog)[select,]</span>
<span id="cb9-26"><a href="#cb9-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-27"><a href="#cb9-27" aria-hidden="true" tabindex="-1"></a>data_mod <span class="ot">&lt;-</span> mat[<span class="fu">rownames</span>(mat) <span class="sc">%in%</span> rows, ] </span>
<span id="cb9-28"><a href="#cb9-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-29"><a href="#cb9-29" aria-hidden="true" tabindex="-1"></a>matrownames_data_mod <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">rownames</span>(data_mod))</span>
<span id="cb9-30"><a href="#cb9-30" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-31"><a href="#cb9-31" aria-hidden="true" tabindex="-1"></a>matrownames_data_mod <span class="ot">&lt;-</span> matrownames_data_mod <span class="sc">%&gt;%</span> <span class="fu">separate</span>(rownames.data_mod., <span class="fu">c</span>(<span class="st">&quot;id&quot;</span>, <span class="st">&quot;symbol&quot;</span>),<span class="at">sep =</span><span class="st">&quot;([|])&quot;</span>)</span>
<span id="cb9-32"><a href="#cb9-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-33"><a href="#cb9-33" aria-hidden="true" tabindex="-1"></a>select2 <span class="ot">&lt;-</span> <span class="fu">order</span>(<span class="fu">rowMeans</span>(<span class="fu">counts</span>(dds,<span class="at">normalized=</span><span class="cn">TRUE</span>)),</span>
<span id="cb9-34"><a href="#cb9-34" aria-hidden="true" tabindex="-1"></a>                <span class="at">decreasing=</span><span class="cn">TRUE</span>)</span>
<span id="cb9-35"><a href="#cb9-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-36"><a href="#cb9-36" aria-hidden="true" tabindex="-1"></a>mat2 <span class="ot">&lt;-</span> <span class="fu">assay</span>(ddsMat_rlog)[select2,]</span>
<span id="cb9-37"><a href="#cb9-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-38"><a href="#cb9-38" aria-hidden="true" tabindex="-1"></a>data_mod2 <span class="ot">&lt;-</span> mat2[<span class="fu">rownames</span>(mat2) <span class="sc">%in%</span> rows, ] </span>
<span id="cb9-39"><a href="#cb9-39" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-40"><a href="#cb9-40" aria-hidden="true" tabindex="-1"></a>data_mod3 <span class="ot">&lt;-</span> <span class="fu">head</span>(data_mod2, <span class="dv">75</span>)</span>
<span id="cb9-41"><a href="#cb9-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-42"><a href="#cb9-42" aria-hidden="true" tabindex="-1"></a>matrownames_data_mod3 <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">rownames</span>(data_mod3))</span>
<span id="cb9-43"><a href="#cb9-43" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-44"><a href="#cb9-44" aria-hidden="true" tabindex="-1"></a>matrownames_data_mod3 <span class="ot">&lt;-</span> matrownames_data_mod3 <span class="sc">%&gt;%</span> <span class="fu">separate</span>(rownames.data_mod3., <span class="fu">c</span>(<span class="st">&quot;id&quot;</span>, <span class="st">&quot;symbol&quot;</span>),<span class="at">sep =</span><span class="st">&quot;([|])&quot;</span>)</span>
<span id="cb9-45"><a href="#cb9-45" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-46"><a href="#cb9-46" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-47"><a href="#cb9-47" aria-hidden="true" tabindex="-1"></a>samples <span class="ot">&lt;-</span> <span class="fu">as.tibble</span>(<span class="fu">read.csv</span>(<span class="st">&quot;data/sample_info_CIVSA_DAvsnDA.csv&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;,&quot;</span>))</span>
<span id="cb9-48"><a href="#cb9-48" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-49"><a href="#cb9-49" aria-hidden="true" tabindex="-1"></a>annot_col <span class="ot">&lt;-</span> samples <span class="sc">%&gt;%</span></span>
<span id="cb9-50"><a href="#cb9-50" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_to_rownames</span>(<span class="st">&#39;samplename&#39;</span>) <span class="sc">%&gt;%</span></span>
<span id="cb9-51"><a href="#cb9-51" aria-hidden="true" tabindex="-1"></a>  <span class="fu">as.data.frame</span>()</span>
<span id="cb9-52"><a href="#cb9-52" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-53"><a href="#cb9-53" aria-hidden="true" tabindex="-1"></a>ann_colors <span class="ot">=</span> <span class="fu">list</span>(</span>
<span id="cb9-54"><a href="#cb9-54" aria-hidden="true" tabindex="-1"></a>  <span class="at">condition =</span> <span class="fu">c</span>(<span class="at">nDA =</span> <span class="st">&quot;#FF9289&quot;</span>, <span class="at">DA =</span> <span class="st">&quot;#00DAE0&quot;</span>))</span></code></pre></div>
<div id="unsupervised-clustering" class="section level3" number="4.2.1">
<h3><span class="header-section-number">4.2.1</span> Unsupervised
Clustering</h3>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a>pheatmap<span class="sc">::</span><span class="fu">pheatmap</span>(</span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">mat =</span> data_mod,</span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">color =</span> <span class="fu">colorRampPalette</span>(<span class="fu">rev</span>(<span class="fu">brewer.pal</span>(<span class="dv">11</span>, <span class="st">&quot;RdYlBu&quot;</span>)))(<span class="dv">155</span>),</span>
<span id="cb10-4"><a href="#cb10-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">scale =</span> <span class="st">&quot;row&quot;</span>,</span>
<span id="cb10-5"><a href="#cb10-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">annotation_col =</span> annot_col,</span>
<span id="cb10-6"><a href="#cb10-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">annotation_colors =</span> ann_colors,</span>
<span id="cb10-7"><a href="#cb10-7" aria-hidden="true" tabindex="-1"></a>  <span class="at">fontsize =</span><span class="dv">10</span>,</span>
<span id="cb10-8"><a href="#cb10-8" aria-hidden="true" tabindex="-1"></a>  <span class="at">show_colnames =</span> F,</span>
<span id="cb10-9"><a href="#cb10-9" aria-hidden="true" tabindex="-1"></a>  <span class="at">cluster_rows =</span> T,</span>
<span id="cb10-10"><a href="#cb10-10" aria-hidden="true" tabindex="-1"></a>  <span class="at">cluster_cols =</span> T,</span>
<span id="cb10-11"><a href="#cb10-11" aria-hidden="true" tabindex="-1"></a>  <span class="at">annotation_legend =</span> T,</span>
<span id="cb10-12"><a href="#cb10-12" aria-hidden="true" tabindex="-1"></a>  <span class="at">legend =</span> T,</span>
<span id="cb10-13"><a href="#cb10-13" aria-hidden="true" tabindex="-1"></a>  <span class="at">show_rownames =</span> F,</span>
<span id="cb10-14"><a href="#cb10-14" aria-hidden="true" tabindex="-1"></a>  <span class="at">clustering_distance_rows =</span> <span class="st">&quot;correlation&quot;</span>,</span>
<span id="cb10-15"><a href="#cb10-15" aria-hidden="true" tabindex="-1"></a>  <span class="at">clustering_method =</span> <span class="st">&quot;ward.D&quot;</span>,</span>
<span id="cb10-16"><a href="#cb10-16" aria-hidden="true" tabindex="-1"></a>  <span class="at">main =</span> <span class="st">&quot;Heatmap of Significant DEGs -- Unsupervised Clustering&quot;</span>,</span>
<span id="cb10-17"><a href="#cb10-17" aria-hidden="true" tabindex="-1"></a>  <span class="at">cluster_row_slices =</span> <span class="cn">TRUE</span>,</span>
<span id="cb10-18"><a href="#cb10-18" aria-hidden="true" tabindex="-1"></a>  <span class="at">show_row_dend =</span> <span class="cn">FALSE</span>,</span>
<span id="cb10-19"><a href="#cb10-19" aria-hidden="true" tabindex="-1"></a>  <span class="at">cutree_rows =</span> <span class="dv">4</span>,</span>
<span id="cb10-20"><a href="#cb10-20" aria-hidden="true" tabindex="-1"></a>  <span class="at">cutree_cols =</span> <span class="dv">2</span>,</span>
<span id="cb10-21"><a href="#cb10-21" aria-hidden="true" tabindex="-1"></a>  <span class="at">treeheight_row =</span> <span class="dv">10</span>,</span>
<span id="cb10-22"><a href="#cb10-22" aria-hidden="true" tabindex="-1"></a>  <span class="at">treeheight_col =</span> <span class="dv">10</span>,</span>
<span id="cb10-23"><a href="#cb10-23" aria-hidden="true" tabindex="-1"></a>  <span class="co">#filename = &quot;plots/DAvsnDA_HeatmapDEGs.pdf&quot;,</span></span>
<span id="cb10-24"><a href="#cb10-24" aria-hidden="true" tabindex="-1"></a>  <span class="at">legend_labels =</span> <span class="fu">c</span>(<span class="st">&quot;Da&quot;</span>, <span class="st">&quot;nDA&quot;</span>)</span>
<span id="cb10-25"><a href="#cb10-25" aria-hidden="true" tabindex="-1"></a>)</span></code></pre></div>
<p><img src="figure/DAvsnDA.Rmd/Filter%20Heatmaps%20by%20DEGs2%20Unsupervised-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Filter-Heatmaps-by-DEGs2-Unsupervised-1">
Past versions of "Filter Heatmaps by DEGs2 Unsupervised-1.png"
</button>
</p>
<div id="fig-no-spaces-Filter-Heatmaps-by-DEGs2-Unsupervised-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/bfc97c14694a0cfb3bb5699555e3e157ce3f3035/docs/figure/DAvsnDA.Rmd/Filter Heatmaps by DEGs2 Unsupervised-1.png" target="_blank">bfc97c1</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="specified-genes" class="section level3" number="4.2.2">
<h3><span class="header-section-number">4.2.2</span> Specified
Genes</h3>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a>pheatmap<span class="sc">::</span><span class="fu">pheatmap</span>(</span>
<span id="cb11-2"><a href="#cb11-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">mat =</span> data_mod3,</span>
<span id="cb11-3"><a href="#cb11-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">color =</span> <span class="fu">colorRampPalette</span>(<span class="fu">rev</span>(<span class="fu">brewer.pal</span>(<span class="dv">11</span>, <span class="st">&quot;RdYlBu&quot;</span>)))(<span class="dv">155</span>),</span>
<span id="cb11-4"><a href="#cb11-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">scale =</span> <span class="st">&quot;row&quot;</span>, </span>
<span id="cb11-5"><a href="#cb11-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">annotation_col =</span> annot_col,</span>
<span id="cb11-6"><a href="#cb11-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">annotation_colors =</span> ann_colors,</span>
<span id="cb11-7"><a href="#cb11-7" aria-hidden="true" tabindex="-1"></a>  <span class="at">fontsize =</span> <span class="dv">6</span>, </span>
<span id="cb11-8"><a href="#cb11-8" aria-hidden="true" tabindex="-1"></a>  <span class="at">show_colnames =</span> T,</span>
<span id="cb11-9"><a href="#cb11-9" aria-hidden="true" tabindex="-1"></a>  <span class="at">cluster_rows =</span> T,</span>
<span id="cb11-10"><a href="#cb11-10" aria-hidden="true" tabindex="-1"></a>  <span class="at">cluster_cols =</span> T,</span>
<span id="cb11-11"><a href="#cb11-11" aria-hidden="true" tabindex="-1"></a>  <span class="at">annotation_legend =</span> T,</span>
<span id="cb11-12"><a href="#cb11-12" aria-hidden="true" tabindex="-1"></a>  <span class="at">legend =</span> T,</span>
<span id="cb11-13"><a href="#cb11-13" aria-hidden="true" tabindex="-1"></a>  <span class="at">show_rownames =</span> T,</span>
<span id="cb11-14"><a href="#cb11-14" aria-hidden="true" tabindex="-1"></a>  <span class="at">clustering_distance_rows =</span> <span class="st">&quot;correlation&quot;</span>,</span>
<span id="cb11-15"><a href="#cb11-15" aria-hidden="true" tabindex="-1"></a>  <span class="at">clustering_method =</span> <span class="st">&quot;ward.D&quot;</span>,</span>
<span id="cb11-16"><a href="#cb11-16" aria-hidden="true" tabindex="-1"></a>  <span class="at">main =</span> <span class="st">&quot;Heatmap of Significant DEGs -- Supervised Clustering&quot;</span>,</span>
<span id="cb11-17"><a href="#cb11-17" aria-hidden="true" tabindex="-1"></a>  <span class="at">cluster_row_slices =</span> <span class="cn">TRUE</span>,</span>
<span id="cb11-18"><a href="#cb11-18" aria-hidden="true" tabindex="-1"></a>  <span class="at">show_row_dend =</span> <span class="cn">FALSE</span>,</span>
<span id="cb11-19"><a href="#cb11-19" aria-hidden="true" tabindex="-1"></a>  <span class="at">cutree_rows =</span> <span class="dv">4</span>,</span>
<span id="cb11-20"><a href="#cb11-20" aria-hidden="true" tabindex="-1"></a>  <span class="at">cutree_cols =</span> <span class="dv">3</span>,</span>
<span id="cb11-21"><a href="#cb11-21" aria-hidden="true" tabindex="-1"></a>  <span class="at">gaps_col =</span> <span class="fu">c</span>(<span class="dv">4</span>, <span class="dv">7</span>),</span>
<span id="cb11-22"><a href="#cb11-22" aria-hidden="true" tabindex="-1"></a>  <span class="at">treeheight_row =</span> <span class="dv">10</span>,</span>
<span id="cb11-23"><a href="#cb11-23" aria-hidden="true" tabindex="-1"></a>  <span class="at">treeheight_col =</span> <span class="dv">10</span>,</span>
<span id="cb11-24"><a href="#cb11-24" aria-hidden="true" tabindex="-1"></a>  <span class="co">#filename = &quot;plots/DAvsnDA_HeatmapDEGs_withLabs.pdf&quot;,</span></span>
<span id="cb11-25"><a href="#cb11-25" aria-hidden="true" tabindex="-1"></a>  <span class="at">legend_labels =</span> <span class="fu">c</span>(<span class="st">&quot;DA&quot;</span>, <span class="st">&quot;nDA&quot;</span>),</span>
<span id="cb11-26"><a href="#cb11-26" aria-hidden="true" tabindex="-1"></a>  <span class="at">labels_row =</span> matrownames_data_mod3<span class="sc">$</span>symbol</span>
<span id="cb11-27"><a href="#cb11-27" aria-hidden="true" tabindex="-1"></a>)</span></code></pre></div>
<p><img src="figure/DAvsnDA.Rmd/Filter%20Heatmaps%20by%20DEGs2-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Filter-Heatmaps-by-DEGs2-1">
Past versions of "Filter Heatmaps by DEGs2-1.png"
</button>
</p>
<div id="fig-no-spaces-Filter-Heatmaps-by-DEGs2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/1ee1d086cd39bf3115343f820fe5f9cd5de1d6a1/docs/figure/DAvsnDA.Rmd/Filter Heatmaps by DEGs2-1.png" target="_blank">1ee1d08</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="volcano-plots" class="section level2" number="4.3">
<h2><span class="header-section-number">4.3</span> Volcano Plots</h2>
<div id="visualize-results-volcano-plots-non-interactive"
class="section level3" number="4.3.1">
<h3><span class="header-section-number">4.3.1</span> Visualize Results:
Volcano Plots (Non-Interactive)</h3>
<p>Volcano plots are generated by gathering the FDR corrected p-values
from each analysis. The adjusted p-values undergo a -log10
transformation to generate these FDR corrected values. Any rows
containing “NA” are ommitted from analysis. Data points are colored
based on increasing or decreasing value and plotted using ggplot2.
Adjusted p-value cutoff is 1.3 following log transformation.(-log10 0.05
= 1.3)</p>
<p><img src="figure/DAvsnDA.Rmd/Volcano%20Plots-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Volcano-Plots-1">
Past versions of "Volcano Plots-1.png"
</button>
</p>
<div id="fig-no-spaces-Volcano-Plots-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/7c1df5e6061f4671761a1740a2132edfc3958596/docs/figure/DAvsnDA.Rmd/Volcano Plots-1.png" target="_blank">7c1df5e</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/c31538bbb33383ec16632a62b439809eb31d4b2e/docs/figure/DAvsnDA.Rmd/Volcano Plots-1.png" target="_blank">c31538b</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/bfc97c14694a0cfb3bb5699555e3e157ce3f3035/docs/figure/DAvsnDA.Rmd/Volcano Plots-1.png" target="_blank">bfc97c1</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/avm27/CocaineIVSA_RNASequencing_mDANeurons/blob/1ee1d086cd39bf3115343f820fe5f9cd5de1d6a1/docs/figure/DAvsnDA.Rmd/Volcano Plots-1.png" target="_blank">1ee1d08</a>
</td>
<td>
avm27
</td>
<td>
2024-07-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.2.1 (2022-06-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 22.04.4 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.10.0
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.10.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] ggrepel_0.9.3                            
 [2] BiocParallel_1.32.6                      
 [3] conflicted_1.2.0                         
 [4] VennDiagram_1.7.3                        
 [5] futile.logger_1.4.3                      
 [6] regionReport_1.32.0                      
 [7] kableExtra_1.3.4                         
 [8] vidger_1.18.0                            
 [9] tables_0.9.17                            
[10] knitr_1.43                               
[11] Glimma_2.8.0                             
[12] Mus.musculus_1.3.1                       
[13] TxDb.Mmusculus.UCSC.mm10.knownGene_3.10.0
[14] OrganismDbi_1.40.0                       
[15] GenomicFeatures_1.50.4                   
[16] gplots_3.1.3                             
[17] lubridate_1.9.2                          
[18] forcats_1.0.0                            
[19] stringr_1.5.1                            
[20] purrr_1.0.2                              
[21] readr_2.1.4                              
[22] tibble_3.2.1                             
[23] tidyverse_2.0.0                          
[24] statmod_1.5.0                            
[25] tweeDEseqCountData_1.36.0                
[26] edgeR_3.40.2                             
[27] limma_3.54.2                             
[28] GOSemSim_2.24.0                          
[29] apeglm_1.20.0                            
[30] ggpubr_0.6.0                             
[31] SPIA_2.50.0                              
[32] KEGGgraph_1.58.3                         
[33] ggnewscale_0.4.9                         
[34] enrichplot_1.18.4                        
[35] tidyr_1.3.1                              
[36] WGCNA_1.72-1                             
[37] fastcluster_1.2.3                        
[38] dynamicTreeCut_1.63-1                    
[39] pathview_1.38.0                          
[40] gage_2.48.0                              
[41] dplyr_1.1.4                              
[42] topGO_2.50.0                             
[43] SparseM_1.81                             
[44] graph_1.76.0                             
[45] GO.db_3.16.0                             
[46] RColorBrewer_1.1-3                       
[47] genefilter_1.80.3                        
[48] pheatmap_1.0.12                          
[49] ggsci_3.0.0                              
[50] tximport_1.26.1                          
[51] org.Mm.eg.db_3.16.0                      
[52] AnnotationDbi_1.60.2                     
[53] DOSE_3.24.2                              
[54] ReactomePA_1.42.0                        
[55] biomaRt_2.54.1                           
[56] clusterProfiler_4.6.2                    
[57] ggplot2_3.5.0.9000                       
[58] DESeq2_1.38.3                            
[59] SummarizedExperiment_1.28.0              
[60] Biobase_2.58.0                           
[61] MatrixGenerics_1.10.0                    
[62] matrixStats_1.0.0                        
[63] GenomicRanges_1.50.2                     
[64] GenomeInfoDb_1.34.9                      
[65] IRanges_2.32.0                           
[66] S4Vectors_0.36.2                         
[67] BiocGenerics_0.44.0                      
[68] gprofiler2_0.2.2                         
[69] workflowr_1.7.0                          

loaded via a namespace (and not attached):
  [1] Hmisc_5.1-0              svglite_2.1.1            ps_1.7.5                
  [4] Rsamtools_2.14.0         foreach_1.5.2            rprojroot_2.0.3         
  [7] crayon_1.5.2             MASS_7.3-60              nlme_3.1-162            
 [10] backports_1.4.1          impute_1.72.3            rlang_1.1.3             
 [13] XVector_0.38.0           HDO.db_0.99.1            callr_3.7.3             
 [16] filelock_1.0.2           rjson_0.2.21             bit64_4.0.5             
 [19] glue_1.7.0               rngtools_1.5.2           parallel_4.2.1          
 [22] processx_3.8.2           DEFormats_1.26.0         tidyselect_1.2.1        
 [25] XML_3.99-0.14            GenomicAlignments_1.34.1 xtable_1.8-4            
 [28] magrittr_2.0.3           evaluate_0.21            bibtex_0.5.1            
 [31] cli_3.6.2                zlibbioc_1.44.0          doRNG_1.8.6             
 [34] rstudioapi_0.14          whisker_0.4.1            bslib_0.5.0             
 [37] rpart_4.1.19             derfinderHelper_1.32.0   fastmatch_1.1-3         
 [40] BiocStyle_2.26.0         lambda.r_1.2.4           treeio_1.22.0           
 [43] xfun_0.39                gson_0.1.0               cluster_2.1.4           
 [46] caTools_1.18.2           tidygraph_1.2.3          KEGGREST_1.38.0         
 [49] ape_5.7-1                Biostrings_2.66.0        png_0.1-8               
 [52] reshape_0.8.9            withr_3.0.0              bitops_1.0-7            
 [55] ggforce_0.4.1            RBGL_1.74.0              plyr_1.8.8              
 [58] coda_0.19-4              bumphunter_1.40.0        pillar_1.9.0            
 [61] cachem_1.0.8             fs_1.6.2                 graphite_1.44.0         
 [64] vctrs_0.6.5              generics_0.1.3           tools_4.2.1             
 [67] foreign_0.8-84           munsell_0.5.1            tweenr_2.0.2            
 [70] fgsea_1.24.0             DelayedArray_0.24.0      fastmap_1.1.1           
 [73] compiler_4.2.1           abind_1.4-5              httpuv_1.6.11           
 [76] rtracklayer_1.58.0       plotly_4.10.2            GenomeInfoDbData_1.2.9  
 [79] gridExtra_2.3            lattice_0.21-8           utf8_1.2.4              
 [82] later_1.3.1              BiocFileCache_2.6.1      jsonlite_1.8.7          
 [85] GGally_2.1.2             scales_1.3.0             tidytree_0.4.2          
 [88] carData_3.0-5            lazyeval_0.2.2           promises_1.2.0.1        
 [91] car_3.1-2                doParallel_1.0.17        checkmate_2.2.0         
 [94] rmarkdown_2.23           cowplot_1.1.1            webshot_0.5.5           
 [97] downloader_0.4           BSgenome_1.66.3          igraph_1.5.0            
[100] survival_3.5-5           numDeriv_2016.8-1.1      yaml_2.3.7              
[103] systemfonts_1.0.4        htmltools_0.5.5          memoise_2.0.1           
[106] VariantAnnotation_1.44.1 BiocIO_1.8.0             locfit_1.5-9.8          
[109] graphlayouts_1.0.0       viridisLite_0.4.2        digest_0.6.32           
[112] rappdirs_0.3.3           knitrBootstrap_1.0.2     futile.options_1.0.1    
[115] emdbook_1.3.13           RSQLite_2.3.1            yulab.utils_0.0.6       
[118] derfinder_1.32.0         data.table_1.14.8        blob_1.2.4              
[121] preprocessCore_1.60.2    labeling_0.4.3           splines_4.2.1           
[124] Formula_1.2-5            RCurl_1.98-1.12          broom_1.0.5             
[127] hms_1.1.3                colorspace_2.1-0         base64enc_0.1-3         
[130] BiocManager_1.30.21      aplot_0.1.10             nnet_7.3-19             
[133] sass_0.4.6               Rcpp_1.0.10              mvtnorm_1.2-2           
[136] fansi_1.0.6              tzdb_0.4.0               R6_2.5.1                
[139] lifecycle_1.0.4          formatR_1.14             curl_5.0.1              
[142] ggsignif_0.6.4           jquerylib_0.1.4          Matrix_1.5-4.1          
[145] qvalue_2.30.0            org.Hs.eg.db_3.16.0      iterators_1.0.14        
[148] RefManageR_1.4.0         htmlwidgets_1.6.2        markdown_1.7            
[151] polyclip_1.10-4          shadowtext_0.1.2         timechange_0.2.0        
[154] gridGraphics_0.5-1       reactome.db_1.82.0       rvest_1.0.3             
[157] htmlTable_2.4.1          patchwork_1.2.0.9000     bdsmatrix_1.3-6         
[160] codetools_0.2-19         gtools_3.9.4             getPass_0.2-2           
[163] prettyunits_1.1.1        dbplyr_2.3.2             gtable_0.3.5            
[166] DBI_1.1.3                git2r_0.32.0             ggfun_0.1.1             
[169] httr_1.4.6               highr_0.10               KernSmooth_2.23-21      
[172] stringi_1.8.4            progress_1.2.2           reshape2_1.4.4          
[175] farver_2.1.2             annotate_1.76.0          viridis_0.6.3           
[178] Rgraphviz_2.42.0         ggtree_3.6.2             xml2_1.3.4              
[181] bbmle_1.0.25             restfulr_0.0.15          geneplotter_1.76.0      
[184] ggplotify_0.1.1          bit_4.0.5                scatterpie_0.2.1        
[187] ggraph_2.1.0             pkgconfig_2.0.3          rstatix_0.7.2           
[190] GenomicFiles_1.34.0     </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>





</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
